src/cli/util.ts

```diff
@@ -55,6 +55,46 @@ export function renderInstallCommand(dir: string, module: string): string {
   return `npm install --save --save-dev -f --no-package-lock --prefix="${dir}" ${module}`;
 }
 
+export function moduleExists(cwd: string, module: string): boolean {
+  try {
+    logging.info(`Validating external module's (${module}) existence...`);
```

Comment:
I don't think this needs to be visible in the regular logs.
```suggestion
    logging.debug(`Checking if external module '${module}' exists...`);
```

And maybe also add a debug log for the various result cases.